src/System.IO.FileSystem/tests/FileInfo/Open.cs

```diff
@@ -12,6 +12,40 @@ protected override FileStream CreateFileStream(string path, FileMode mode)
         {
             return new FileInfo(path).Open(mode);
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "FileInfo.Open(string, filemode) on netfx always uses FileAccess.ReadWrite instead of choosing a FileAccess based on the FileMode. This bug was fixed in netcoreapp.")]
```

Comment:
I agree, it's not worth creating issues just to track ports. We've made very many fixes to Core already  that we haven't tracked so when we get to that point we couldn't rely on issues anyway. Exception if it's something critical enough we might service desktop for it. Then I'd email Alex G.